docs/docs/genai/eval-monitor/scorers/third-party/ragas.mdx

```diff
@@ -0,0 +1,151 @@
+| <APILink fn="mlflow.genai.scorers.ragas.StringPresence">StringPresence</APILink>                 | Is a specific string present in the output?                  | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/traditional/#string-presence)           |
+| <APILink fn="mlflow.genai.scorers.ragas.ExactMatch">ExactMatch</APILink>                         | Does output exactly match expected output?                   | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/traditional/#exact-match)               |
+
+### General Purpose
+
+Flexible evaluation metrics for various use cases:
+
+| Scorer                                                                             | What does it evaluate?                             | RAGAS Docs                                                                                                                             |
+| ---------------------------------------------------------------------------------- | -------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| <APILink fn="mlflow.genai.scorers.ragas.AspectCritic">AspectCritic</APILink>       | Evaluates specific aspects of the output using LLM | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/aspect_critic/)                                              |
+| <APILink fn="mlflow.genai.scorers.ragas.RubricsScore">RubricsScore</APILink>       | Scores output based on predefined rubrics          | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/general_purpose/#rubrics-based-criteria-scoring)             |
+| <APILink fn="mlflow.genai.scorers.ragas.InstanceRubrics">InstanceRubrics</APILink> | Scores output based on instance-specific rubrics   | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/general_purpose/#instance-specific-rubrics-criteria-scoring) |
+
+### Other Tasks
+
+Specialized metrics for specific tasks:
+
+| Scorer                                                                                   | What does it evaluate?        | RAGAS Docs                                                                                      |
+| ---------------------------------------------------------------------------------------- | ----------------------------- | ----------------------------------------------------------------------------------------------- |
+| <APILink fn="mlflow.genai.scorers.ragas.SummarizationScore">SummarizationScore</APILink> | Quality of text summarization | [Link](https://docs.ragas.io/en/stable/concepts/metrics/available_metrics/summarization_score/) |
+
+## Creating Scorers by Name
+
+If a particular RAGAS metric is not listed above, you can create it dynamically using <APILink fn="mlflow.genai.scorers.ragas.get_scorer">get_scorer</APILink>:
```

Comment:
```suggestion
If a particular RAGAS metric is not listed above, you can create it dynamically using <APILink fn="mlflow.genai.scorers.ragas.get_scorer">mlflow.genai.scorers.ragas.get_scorer</APILink>:
```

Unless they click the link, users may think they can call `mlflow.genai.scorers.get_scorer`.

Reply:
hmm I'm probably just being dumb but why is that/why is this an issue? there's a code example of how to use it right after

Reply:
oh no you're right --the code example makes this clear. Ignore me :)